doc/source/whatsnew/v2.0.0.rst

```diff
@@ -557,6 +557,7 @@ Removal of prior version deprecations/changes
 - Enforced deprecation ``numeric_only=None`` (the default) in DataFrame reductions that would silently drop columns that raised; ``numeric_only`` now defaults to ``False`` (:issue:`41480`)
 - Changed default of ``numeric_only`` to ``False`` in all DataFrame methods with that argument (:issue:`46096`, :issue:`46906`)
 - Changed default of ``numeric_only`` to ``False`` in :meth:`Series.rank` (:issue:`47561`)
+- Enforced deprecation of silently dropping nuisance columns in groupby and resample operations when ``numeric_only=False`` (:issue:`41475`)
```

Comment:
maybe this is different from the issue i recall, but i thought the silent-dropping was with numeric_only=None, and we're just getting rid of that altogether?

Reply:
Right - I've been trying to figure out how to split this up into manageable pieces. Certain ops in 1.x would drop nuisance columns even when numeric_only=False, and others would only do it when numeric_only=None. It seemed easiest to handle the False case first, then get rid of the None.

Reply:
cool, thanks